cfgov/tccp/jinja2/tccp/landing_page.html

```diff
@@ -23,6 +23,18 @@ <h1>{{ heading }}</h1>
         </p>
     </div>
 
+    <div class="block block__sub">
+        <h2>
```

Comment:
```suggestion
        <h2 class="h3">
```

Looks like this is styled as an `h3` in the latest mockups.